src/__generated__/ShippingAndPaymentSummary_order.graphql.ts

```diff
@@ -5,6 +5,7 @@ import { ShippingAddress_ship$ref } from "./ShippingAddress_ship.graphql";
 declare const _ShippingAndPaymentSummary_order$ref: unique symbol;
 export type ShippingAndPaymentSummary_order$ref = typeof _ShippingAndPaymentSummary_order$ref;
 export type ShippingAndPaymentSummary_order = {
+    readonly state: string | null;
```

Comment:
I’m really surprised that [this isn’t an enum](https://github.com/artsy/metaphysics/blob/5e70fdf813d582c54d70c073ad9a25d8e9abc585/src/schema/ecommerce/types/order.ts#L27-L30). Isn’t the possible states finite and known? If so, I’d urge y’all to update that before this gets shipped.

Reply:
I 1000% agree. It actually _is_ an [enum in exchange](https://github.com/artsy/exchange/blob/014227c476f3d78283495bcfa85ec2ebd8a00087/_schema.graphql#L260-L281). Way too many string references that aren't tied to types used in this code currently... makes me a bit nervous. 😅